frontend/integration-tests/views/service-catalog.view.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-undef, no-unused-vars */
```

Comment:
For items that are not specific to service catalog, but are common to other details / lists pages, we should either use what's in crud.view.ts or add things that are missing. This file should only have service catalog specific items.